src/AssembleMomentumEdgeSolverAlgorithm.C

```diff
@@ -13,14 +13,19 @@
 #include <LinearSystem.h>
 #include <PecletFunction.h>
 #include <Realm.h>
-
+#include <SolutionOptions.h>
```

Comment:
I think that we want a separate class for the perturbation approach. Otherwise, for a non-perturbation approach, this affects the baseline laminar and turbulent capability.